src/collective/fingerpointing/logger.py

```diff
@@ -1,5 +1,55 @@
 # -*- coding: utf-8 -*-
-from collective.fingerpointing.config import PROJECTNAME
 import logging
+import time
+
+import zc.lockfile
+from collective.fingerpointing.config import PROJECTNAME
```

Comment:
please follow the convention on [grouping and sorting imports](http://docs.plone.org/develop/styleguide/python.html#grouping-and-sorting)